src/lavinmq/http/controller/prometheus.cr

```diff
@@ -264,6 +264,10 @@ module LavinMQ
                       value: @amqp_server.followers.size,
                       type:  "gauge",
                       help:  "Amount of follower nodes connected"})
+        writer.write({name:  "clustering_max_lag",
+                      value: Config.instance.clustering_max_lag,
+                      type:  "gauge",
+                      help:  "Max unsynced replicated messages"})
```

Comment:
WDYT?

```suggestion
                      help:  "Maximum allowed unsynced replicated messages"})
```

Also, here it says "actions" https://github.com/cloudamqp/lavinmq/blob/a05fb6b28332776b66d2129f8f9ce7750e2a1532/src/lavinmq/config.cr#L58, not sure if we want to be consistent :)

Reply:
I just copied the text from here https://github.com/cloudamqp/lavinmq/blob/main/src/lavinmq/config.cr#L136 🙈 
But yeah, we should be consistent!

Reply:
Maybe easiest to update the comment then, hehe :)

Reply:
Yeah :) 
Hmm, this all might be a little confusing. `clustering_max_lag` is the maximum number of actions that a follower can lag behind, but the `lag` we show in GUI/metrics is how many actual bytes the follower is lagging behind. So the numbers don't really have much to do with each other... 🤔

Reply:
Agree that it can be confusing. Could we name it something like max_allowed_cluster_actions, and let lag be lag_bytes?

Reply:
Yeah, it might be reasonable to rename at least one of them. 
Maybe rename 
`lag` -> `lag_in_bytes`, 
`max_lag` -> `max_lag_in_actions`, 
and add `lag_in_actions` 🤔

Reply:
Here's PRs for the changes mentioned in this thread
https://github.com/cloudamqp/lavinmq/pull/810
https://github.com/cloudamqp/lavinmq/pull/811